client/site-profiler/components/site-profiler-v2.tsx

```diff
@@ -39,10 +39,11 @@ import './styles-v2.scss';
 interface Props {
 	routerDomain?: string;
 	hash?: string;
+	routerRef?: string;
```

Comment:
Non-blocking and very nit-pick 😛 but, the name of the `prop` seems to be related to a `React` ref (that's the first thing I thought). Do you think it could be changed to something like `routerOrigin` or another synonym?

Reply:
Yep, that's a good idea!